### PR TITLE
bugfix: S3C-3810 Skip headers on 304 response

### PR DESCRIPTION
**Pull request by miniscruff**

### CODE REVIEW

File: lib/s3routes/routesUtils.js [hunk 1]
```diff
@@ -85,8 +85,17 @@ const XMLResponseBackend = {
         });
     },
 
-    errorResponse: function errorXMLResponse(errCode, response, log,
-                                             corsHeaders) {
+    errorResponse: function errorXMLResponse(errCode, response, log, corsHeaders) {
+        setCommonResponseHeaders(corsHeaders, response, log);
+        if (errCode.code === 304) {
```
Comment (naren-rajendran) [inline lib/s3routes/routesUtils.js:91]: may be add a comment why we are doing this!

**Review by naren-rajendran — APPROVED** [review]

**Review by mrepop — APPROVED** [review]

File: lib/s3routes/routesUtils.js (hunk 1 above)
Comment (saitharunthodupunuru) [reply lib/s3routes/routesUtils.js:91]: I'd recommend adding the JIRA ticket number of the bugfix too in your comment like 
`// [S3C-3810] ....reason...`

**Review by saitharunthodupunuru — APPROVED** [review]

**Pull request merged**